src/components/comments/CommentSection.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Question from "./Question.js";
+import Answer from "./Answer.js";
+export default function CommentSection(props) {
+  let followupSection = [];
+  for (let i = 0; i < props.followups.length; i++) {
+    if (props.followups[i].question != null) {
+      followupSection.push(
+        <Question questionText={props.followups[i].question} />
+      );
+      followupSection.push(<div style={styles.separator}></div>);
+    }
+    if (
+      props.followups[i].answer != null ||
```

Comment:
Is this supposed to be `||`? You'll push `null` if `answer` is `null` but `answerHeader` isn't?

Reply:
Yup, since some answers will just be bold text with no regular text underneath